tests/cases/fourslash/server/autoImportProvider_importsMap1.ts

```diff
@@ -29,4 +29,4 @@
 // @Filename: /src/a.ts
 //// isBrowser/**/
 
-verify.importFixModuleSpecifiers("", ["./env/browser.js", "#is-browser"]);
\ No newline at end of file
+verify.importFixModuleSpecifiers("", ["#is-browser", "./env/browser.js"]);
```

Comment:
It should have been in this order due to number of path segments, but `#is-browser` was being demoted because it was being checked against package.json dependencies, as I mentioned in the PR description.